frontend/src/components/productGrid/ModelDropdownComponent.js

```diff
@@ -22,16 +26,22 @@ const ModelDropdownComponent = (props) =>  {
     // set the initial values for the dropdown list derived from getModels
     const variables = getModels(productID);  
     const [ selected, setSelected ] = useState('Select a model');
-
-        return (
-            <div>
-
-                <div className='w3-dropdown-hover w3-green w3-margin-bottom w3-margin-left:5em w3-margin-right w3-margin-top w3-margin-bottom:2em' >
-                <IconArrowDown/>
-                    <button className=' w3-button w3-border w3-white'>{selected}</button>
-                    <div className=' w3-dropdown-content w3-dropdown-hover w3-bar-block w3-border '>
+    //className=' w3-dropdown-hover w3-border w3-white w3-margin-bottom w3-margin-left:5em w3-margin-right w3-margin-top w3-margin-bottom:2em' >
+    return (
+            <Container fluid={true}>
+                <Row className='w3-dropdown-hover w3-margin-left:5em w3-margin-right w3-margin-top w3-margin-bottom:2em'>
+                    <Col xs={2}>  
+                    <button className='w3-button' style={{fontSize:theme.typography.buttontitle.fontSize,fontWeight:theme.typography.buttontitle.fontWeight,lineHeight:theme.typography.buttontitle.lineHeight,letterSpacing:theme.typography.buttonSendtitle.letterSpacing}}>
+                        {selected}
+                    </button>
+                    <div className='w3-dropdown-content w3-bar-block w3-border' style={{fontSize:theme.typography.buttontitle.fontSize,fontWeight:theme.typography.buttontitle.fontWeight,lineHeight:theme.typography.buttontitle.lineHeight,letterSpacing:theme.typography.buttonSendtitle.letterSpacing}}>                       
+                    {/* <button className=' w3-button w3-white' style={{fontSize:theme.typography.buttontitle.fontSize,fontWeight:theme.typography.buttontitle.fontWeight}}>{selected}</button>
+                    
+                    
+                    <div style={{fontSize:theme.typography.buttontitle.fontSize,fontWeight:theme.typography.buttontitle.fontWeight}} className='w3-dropdown-content  w3-white'>
+                    */}
```

Comment:
(You can directly apply a suggestion here from github, just don't forget to pull at your computer afterwards 😉)
```suggestion
```

I like the new Look of the buttons, but I am not 100% set on the gray. What do you think @manifau?